src/course/syllabus/apprenticeship/server-side-app/schedule.md

```diff
@@ -2,13 +2,28 @@
 layout: schedule
 schedule:
   monday:
-    - name: Employer day
+    - name: Validation workshop
+      url: /workshops/form-validation
+      type: workshop
       start: 9:45
       end: 18:00
   tuesday:
+    - name: Project and spike intro
+      start: 12:00
+      end: 12:15
+      type: project
+      url: ../project
+    - name: Project spike
+      start: 12:15
+      end: 13:00
+      url: ../project#spike
```

Comment:
```suggestion
      url: ../project#spikes
```

The link here doesn't work because the title on `/project` is _spikes_. Can either update the link to `../project#spikes` or change the title to _pike_